build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
+  "master": {
+    "nugetPush": [
+        {
+            "kind": "PerBuildPreRelease", 
```

Comment:
Example is right now. If this were in place the entry for dev15.3.x would be Release since we've "shipped".